src/main/java/duke/Tasks/Task.java

```diff
@@ -0,0 +1,31 @@
+package duke.Tasks;
+public class Task {
+    public String task;
+    private boolean done;
```

Comment:
You could consider renaming `done` to `isDone` (or something similar) since it's a `boolean`, as per the [coding standard](https://se-education.org/guides/conventions/java/intermediate.html#naming).